packages/react/src/index.d.ts

```diff
@@ -0,0 +1,1162 @@
+    isActive?: boolean;
+    title?: string;
+    disabled?: boolean;
+  }
+  export interface MultiStepsProps extends BasicProps {
+    current?: number;
+    currentLabel?: string;
+    onChange?: (current: number) => void;
+    direction?: 'horizontal' | 'vertical';
+    variant?:
+      | 'primary'
+      | 'accent'
+      | 'positive'
+      | 'warning'
+      | 'negative'
+      | 'white';
+  }
+  export const MultiSteps: React.FC<MultiStepsProps> & {
+    Item: React.FC<MultiStepsItemProps>;
+  };
+
+  interface OverlayBasicProps extends BasicProps {
+    children: React.ReactNode;
+    popperConfig?: PopperOptions;
+    placement?: PopperPlacement;
+    rootClose?: boolean;
+    rootCloseEvent?: string;
+    rootCloseDisabled?: boolean;
+  }
+  export interface OverlayTriggerProps extends OverlayBasicProps {
+    trigger?: TriggerType | TriggerType[];
+    delay?: number | { show: number; hide: number };
+    hoverOverlay?: boolean;
+    defaultShow?: boolean;
+    overlay: FuncType | React.ReactNode;
+  }
+
+  export interface OverlayProps extends OverlayBasicProps {
+    show?: boolean;
+    target?: React.RefObject<HTMLElement>;
+    container?: HTMLElement;
+    flip?: boolean;
+    containerPadding?: number;
+    onHide?: () => void;
+  }
+  export const Overlay: React.FC<OverlayProps> & {
+    Trigger: React.FC<OverlayTriggerProps>;
+  };
+
+  export interface PageLayoutProps extends BasicWithAsProps {
+    headerProps?: object;
+    footerProps?: object;
+    bodyProps?: object;
+  }
+  export const PageLayout: React.FC<PageLayoutProps> & {
+    Header: React.FC<BasicWithAsProps>;
+    Footer: React.FC<BasicWithAsProps>;
+    Body: React.FC<BasicWithAsProps>;
+  };
+
+  export interface PaginationItemProps extends BasicProps {
+    active?: boolean;
+    disabled?: boolean;
+  }
+  export interface PaginationProps extends BasicProps {
+    sizeControl?: InputSize;
+  }
+  export const Pagination: React.FC<PaginationProps> & {
+    Item: React.FC<PaginationItemProps>;
+  };
+
+  export interface ProblemInfoProps extends BasicProps {
+    topicLabel?: string;
+    topicName?: string;
+    descriptionValue?: string | FuncType;
+    additionalLabel?: string;
+    additionalValue?: string | FuncType;
+    src?: string;
+    onClickImage?: () => void;
+    action?: string | FuncType;
+  }
+  export const ProblemInfo: React.FC<ProblemInfoProps>;
+
+  export interface ProgressProps extends BasicProps {
+    variant?: 'primary' | 'accent' | 'positive' | 'warning' | 'negative';
+    now?: number;
+    height?: number;
+    label?: ReactNodeLike;
+    labelClassName?: string;
+    border?: boolean;
+    striped?: boolean;
+    animated?: boolean;
+  }
+  export const Progress: React.FC<ProgressProps>;
+
+  export interface RatingProps extends BasicProps {
+    disabled?: boolean;
+    emptyIcon?: IconType;
+    readOnly?: boolean;
+    onChange?: (value: number) => void;
+    onChangeActive?: (value: number) => void;
```

Comment:
These two receive `event `as the first argument, ref: https://github.com/tinhvqbk/ahaui/blob/e957aca434578f10a806921a6d065b4fe7393489/packages/react/src/components/Rating/index.js#L104-L108